src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,51 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/add")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        try {
```

Comment:
it's not quite clear what code here, generates exception that you want to catch and redirect user to error page. Let's make try catch block smaller + catching RuntimeException is a bad practice, if you have bad code here, for example NPE or IOB, user will see error page, but it's not his fault. Ideal solution would be custom exception that is related only with invalid entered data. But for this task it would be to complicated so I suggest we just move unrelated code from try block